### PR TITLE
sys-auth/sssd: Enable Kerberos support

### PR DESCRIPTION
**Pull request by mjg59**
Add the appropriate flags for building with Kerberos support and bump the version

### CODE REVIEW

**Comment by mjg59** [comment]
Should fix 1443 and partially fix 968

File: sys-auth/sssd/sssd-1.13.1-r1.ebuild [hunk 1]
```diff
@@ -14,7 +14,7 @@ SRC_URI="http://fedorahosted.org/released/${PN}/${P}.tar.gz"
 
 LICENSE="GPL-3"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~hppa ~ppc ~ppc64 ~x86"
```
Comment (mischief) [inline sys-auth/sssd/sssd-1.13.1-r1.ebuild]: how did this work before if it was ~amd64?

also, this should be 'arm64', arm is 32 bit.

Comment (mjg59) [inline sys-auth/sssd/sssd-1.13.1-r1.ebuild]: Great question, and the answer is I have no idea.

**Comment by mischief** [comment]
lgtm

**Pull request merged**